src/hacbs/components/ApplicationDetails/tabs/overview/visualization/utils/__tests__/visualization-utils.spec.ts

```diff
@@ -130,60 +132,51 @@ describe('workflowToNodes', () => {
   });
 });
 
-describe('sortEnvironments', () => {
-  test('should return empty array for invalid values', () => {
-    expect(sortEnvironments(null)).toHaveLength(0);
-    expect(sortEnvironments(undefined)).toHaveLength(0);
-    expect(sortEnvironments([])).toHaveLength(0);
+describe('getLastEnvironments', () => {
+  test('should return default environment for invalid values', () => {
+    expect(getLastEnvironments([])).toEqual(['static-env']);
```

Comment:
Do you think it is better to test getLastEnvironments([])[0].toBe('static-env') ??

Reply:
yes, toEqual is == and toBe is === . Should we test with toBes instead of toEquals?